services/blockchain-indexer/shared/indexer/transactionProcessor/auth/registerMultisignatureGroup.js

```diff
@@ -30,8 +30,7 @@ const multisignatureIndexSchema = require('../../../database/schema/multisignatu
 
 const getMultisignatureIndex = () => getTableInstance('multisignature', multisignatureIndexSchema, MYSQL_ENDPOINT);
 
-// Command specific constants
-const commandID = '00000000';
+// commandName specific constants
 const commandName = 'registerMultisignatureGroup';
```

Comment:
Please ensure that the comments are appropriate everywhere. Let us be extra careful when using the `Replace All` feature within the code editors.
```suggestion
// Command specific constants
const command = 'registerMultisignatureGroup';
```